### PR TITLE
MBS-8791 Update Release publishing documentation

### PR DESCRIPTION
**Pull request by sergioboi**
extract to separate file
update info about current CI integration builds

### CODE REVIEW

File: docs/content/docs/Release.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Release
+
+{{<avito section>}}
+
+We publish releases to [bintray](https://bintray.com/avito-tech/maven/avito-android).
```
Comment (eugene-krivobokov) [inline docs/content/docs/Release.md]:
```suggestion
We publish releases to [Bintray](https://bintray.com/avito-tech/maven/avito-android).
```

File: docs/content/docs/Release.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Publishing a new release
+
+1. Check current status of [Infra Gradle plugins configuration compatibility with Avito](http://links.k.avito.ru/80)
+    1. If it is `Failed` you could release from previous `Succeed` commits or fix compatibility problems
+1. Check current status of [Nightly Avito integration build](http://links.k.avito.ru/gZ)
+    1. If it is `Failed` you could release from previous `Succeed` commits or fix problems
+1. Checkout a `release branch` with a name equals to `projectVersion`. For example, `2020.9`.\
+This branch must be persistent. It will be used for automation.
```
Comment (eugene-krivobokov) [inline docs/content/docs/Release.md]:
```suggestion
This branch must be persistent. It is used for automation.
```
Not now, but we can skip these details for the sake of explanation.

File: docs/content/docs/Release.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+If you want to run a real CI build against not published release, 
+you need to publish it manually as a temporary version to the internal Artifactory.
+
+## CI integration configurations
+
+- [fast check configuration (internal)](http://links.k.avito.ru/fastCheck) - pull request builds
+- [integration check](http://links.k.avito.ru/ZA) - currently, contains the biggest amount of integration checks
+- [nightly integration check](http://links.k.avito.ru/gZ) - the same as `integration check` but has a bigger amount of emulators
+- [Gradle configuration compatibility check](http://links.k.avito.ru/80) checks the configuration compatibility of our Gradle plugins with Avito repo
```
Comment (eugene-krivobokov) [inline docs/content/docs/Release.md]:
```suggestion
- [Gradle configuration compatibility check](http://links.k.avito.ru/80) - checks the configuration compatibility of our Gradle plugins with Avito repo  
```

File: docs/content/docs/Release.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- To build both projects of special branch, they should have the same name
+
+If you want to run a real CI build against not published release, 
+you need to publish it manually as a temporary version to the internal Artifactory.
+
+## CI integration configurations
+
+- [fast check configuration (internal)](http://links.k.avito.ru/fastCheck) - pull request builds
+- [integration check](http://links.k.avito.ru/ZA) - currently, contains the biggest amount of integration checks
+- [nightly integration check](http://links.k.avito.ru/gZ) - the same as `integration check` but has a bigger amount of emulators
```
Comment (eugene-krivobokov) [inline docs/content/docs/Release.md]:
```suggestion
- [nightly integration check](http://links.k.avito.ru/gZ) - the same as `integration check` but uses more Android emulators
```

Action is better than a state

File: docs/content/docs/Release.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+- If you pick a branch from avito, it will run against develop on github
+- If you pick a branch from github, it will run against develop on avito
+- To build both projects of special branch, they should have the same name
+
+If you want to run a real CI build against not published release, 
+you need to publish it manually as a temporary version to the internal Artifactory.
+
+## CI integration configurations
+
+- [fast check configuration (internal)](http://links.k.avito.ru/fastCheck) - pull request builds
```
Comment (eugene-krivobokov) [inline docs/content/docs/Release.md]:
```suggestion
- [fast check configuration (internal)](http://links.k.avito.ru/fastCheck) - pull request's builds
```

File: docs/content/docs/Release.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+## Publishing a new release
+
+1. Check current status of [Infra Gradle plugins configuration compatibility with Avito](http://links.k.avito.ru/80)
+    1. If it is `Failed` you could release from previous `Succeed` commits or fix compatibility problems
+1. Check current status of [Nightly Avito integration build](http://links.k.avito.ru/gZ)
+    1. If it is `Failed` you could release from previous `Succeed` commits or fix problems
+1. Checkout a `release branch` with a name equals to `projectVersion`. For example, `2020.9`.\
+This branch must be persistent. It will be used for automation.
+1. Manually run [Integration build](http://links.k.avito.ru/ZA) on the `release branch`.
```
Comment (eugene-krivobokov) [inline docs/content/docs/Release.md:15]: Where is publishing to Bintray?
> Manually run [Github publish configuration (internal)](http://links.k.avito.ru/releaseAvitoTools) on the release branch.

Comment (sergioboi) [reply docs/content/docs/Release.md:15]: Missed. I've returned it

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**